typescript/sdk/src/core/HyperlaneCore.ts

```diff
@@ -135,21 +143,24 @@ export class HyperlaneCore extends HyperlaneApp<CoreContracts> {
 
   getDispatchedMessages(sourceTx: ethers.ContractReceipt): DispatchedMessage[] {
     const mailbox = Mailbox__factory.createInterface();
-    const dispatchLogs = sourceTx.logs
-      .map((log) => {
-        try {
-          return mailbox.parseLog(log);
-        } catch (e) {
-          return undefined;
-        }
-      })
-      .filter(
-        (log): log is ethers.utils.LogDescription =>
-          !!log && log.name === 'Dispatch',
-      );
+    const filterLogs = (eventName: string) => {
```

Comment:
Did you maybe plan to use this function more than once, but then realize you didn't need to? Wondering why change this code at all?